Files declared in .angular-cli "assets" are not copied

### Versions
```
Angular CLI: 1.6.4
Node: 8.9.4
OS: win32 x64
```

### Repro steps
My assets are defined as follows in `.angular-cli.json` (pretty much the default configuration):
```
"assets": [
  "assets",
  "favicon.png"
]
```

However, the files are not copied. Neither the assets folder, nor the favicon.
This only arises with @angular/cli@1.6.4. With 1.6.3, everything works fine.

### Mention any other details that might be useful (optional)
Can't really tell you anything else, as no errors or whatsoever are thrown during the build.
The declared assets are just missing in the output directory upon build completion.

I'm experiencing the same issue with version `1.6.5` and Windows 7 x64.

If I try downgrading to version `1.6.3` I get the following error after `ng build --prod`:

```
Error: Cannot find module '@angular-devkit/core'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\PROJECTS\[examples]\PWA\pwa-ng5\node_modules\@angulardevkit\schematics\src\tree\virtual.js:10:16)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
```

Even after installing the package with `npm i -D @angular-devkit/core`, the `ng build --prod` command doesn't copy the assets folder either.

--- a/package.json
+++ b/package.json
@@ -77,7 +77,7 @@
     "clean-css": "^4.1.11",
     "codelyzer": "^4.2.1",
     "conventional-changelog": "^1.1.0",
-    "copy-webpack-plugin": "^4.5.1",
+    "copy-webpack-plugin": "^4.5.2",
     "express": "^4.16.2",
     "file-loader": "^1.1.11",
     "font-awesome": "^4.7.0",

--- a/packages/angular_devkit/build_angular/package.json
+++ b/packages/angular_devkit/build_angular/package.json
@@ -18,7 +18,7 @@
     "autoprefixer": "^8.4.1",
     "circular-dependency-plugin": "^5.0.2",
     "clean-css": "^4.1.11",
-    "copy-webpack-plugin": "^4.5.1",
+    "copy-webpack-plugin": "^4.5.2",
     "file-loader": "^1.1.11",
     "glob": "^7.0.3",
     "html-webpack-plugin": "^3.0.6",

